website/src/content/service-contract.mdx

```diff
@@ -0,0 +1,76 @@
+
+A feature is marked as Beta when is stable enough to be used in production but is not fully feature complete.
+During the Beta period there might still be issues and planned breaking changes. In this phase commercetools will gather the necessary feedback from consumers of the features in order to finalize the missing things and promote the feature to general availability.
+
+# Hosting
+
+Customers using custom applications are responsible for the maintainance of the their custom applications. This includes hosting and deployments.
+commercetools does not take any responsibility towards incidents and service disruptions related to custom applications.
+
+# Releases and breaking changes
+
+Tooling and packages related to custom applications are released and published to [Node Package Manager](https://www.npmjs.com) (NPM) following [Semantic Versioning](https://semver.org/) (SemVer).
+
+Releases are documented in both a `CHANGELOG.md` file and in the GitHub release pages in the following repositories:
+
+- Changelog
+  - [Merchant Center Application Kit](https://github.com/commercetools/merchant-center-application-kit/blob/master/CHANGELOG.md)
+  - [UIKit](https://github.com/commercetools/ui-kit/blob/master/CHANGELOG.md)
+- Release notes
+  - [Merchant Center Application Kit](https://github.com/commercetools/merchant-center-application-kit/releases)
+  - [UIKit](https://github.com/commercetools/ui-kit/releases)
+
+All published packages to NPM are listed in the following package organization:
+
+- [`@commercetools-frontend`](https://www.npmjs.com/org/commercetools-frontend)
+- [`@commercetools-uikit`](https://www.npmjs.com/org/commercetools-uikit)
+
+## Pre-releases and stable releases
+
+By default, all releases go to the `next` distribution tag and should be considered **prereleases**. This gives us a chance to test a release before marking it **stable** in the `latest` distribution channel.
+
+## Breaking changes
+
+Breaking changes are introduced whenever there is a **major** version release, for example from `v1` to `v2`.
+
+> A new major release does not always mean that breaking changes affect consumer of the packages directly.
+> Because of Semantic Versioning, a breaking change can sometimes mean bumping a version of a dependency package
+
+All breaking changes are thoroughly documented in the release notes together with migration steps to faciliate the version update.
+
+# Miminal required maintainance
+
+The Merchant Center applications are always kept up-to-date with the released tools and packages used to build custom applications.
+
+However, customers developing and maintaining custom applications might not always have the time and resources to keep the applications up-to-date.
+commercetools requires that custom applications are at least **maintained at regular periods** and **do not fall back behind too many versions**.
```

Comment:
This whole section, I would like to move up to under the Beta section as it's central. Also, if we can add even some more visual cue to highlight this as a very important section that would be good.

Since it's a front-end library, it would be great if we can clarify why it's important for them to upgrade, i.e. something like "the Merchant Center Application Kit relies on internal Merchant Center internal APIs where there sometimes are breaking changes".

Similarly, we really want to give some guarantees here in order for it to not be completely random. I would preferably say that a major release will have guaranteed support for 12 months during the beta period. If this is too long, let's have a discussion of what period we would agree too, but I think the number of versions is less important than to have a lifecycle support for the major versions.

Reply:
Yeah this section/topic needs to be expanded more. @nkuehn also had some insights about this from the consumer perspective.

We need to provide more specific numbers I suppose, I would set up a meeting with some people to come up with some proposals and a decision.